src/interfaces.ts

```diff
@@ -0,0 +1,15 @@
+export interface ITodo {
```

Comment:
normally interfaces do not start with `I`... change ITodo to `Todo` and other also